apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -62,6 +62,7 @@ private static DatabaseAdminSettings CreateDatabaseAdminSettings()
             internal QueryOptions QueryOptions { get; }
             internal Priority Priority { get; }
             internal string Tag { get; }
+            internal SpannerConversionOptions ConversionOptions { get; }
```

Comment:
Given that we keep the connection anyway, could we maybe change this to:

```csharp
internal SpannerConversionOptions ConversionOptions => SpannerConversionOptions.ForConnection(Connection);
```

... and avoid the implicit field?

Reply:
(That can definitely be done later though.)

Reply:
Done.